lib/Doctrine/ORM/Tools/EntityGenerator.php

```diff
@@ -506,7 +506,7 @@ public function setGenerateAnnotations($bool)
     /**
      * Sets the class fields visibility for the entity (can either be private or protected).
      *
-     * @param bool $visibility
+     * @param string $visibility
```

Comment:
Since only specific constants are allowed, would the use of the psalm enum be a fitting choice?
https://psalm.dev/docs/annotating_code/typing_in_psalm/#specifying-stringint-options-aka-enums

Reply:
Well, I wasn't sure how far to push it for this PR, but if you suggest it, Psalm allows that sort of syntax
`@psalm-param self::FIELD_VISIBLE_*` to restrict usage to existing properties following the pattern. I pushed the change